app/services/api_geo_service.rb

```diff
@@ -79,7 +79,7 @@ def commune_code(departement_code, name)
     private
 
     def communes_by_postal_code_map
-      Rails.cache.fetch('api_geo_communes_v2', expires_in: 1.week) do
+      Rails.cache.fetch('api_geo_communes_v3', expires_in: 1.day) do
```

Comment:
Pour info on peut directement versionner le cache avec `version` : 

```suggestion
      Rails.cache.fetch('api_geo_communes', expires_in: 1.day, version: 3) do
```